modelskill/comparison.py

```diff
@@ -2137,6 +2113,51 @@ def scatter(
         >>> comparer.scatter(model='HKZN_v2', figsize=(10, 10))
         >>> comparer.scatter(observations=['c2','HKNA'])
         """
+
+        # TODO remove in v1.1
```

Comment:
Can _get_deprecated_args not be used here ?

Reply:
Sure